Client/src/Unified/Services/Submitter/Service.cs

```diff
@@ -187,6 +241,133 @@ public string SessionId
+  /// <param name="arguments">List of serialized arguments that will already serialize for MethodName.</param>
+  /// <param name="handler">The handler callBack implemented as IServiceInvocationHandler to get response or result or error</param>
+  /// <returns>Return the taskId string</returns>
+  public IEnumerable<string> Submit(string                    methodName,
+                                    IEnumerable<byte[]>       arguments,
+                                    IServiceInvocationHandler handler)
+  {
+    var armonikPayloads = arguments.Select(args => new ArmonikPayload
+                                                   {
+                                                     ArmonikRequestType  = ArmonikRequestType.Execute,
+                                                     MethodName          = methodName,
+                                                     ClientPayload       = args,
+                                                     SerializedArguments = true,
+                                                   });
+
+    var taskIds   = SessionService.SubmitTasks(armonikPayloads.Select(p => p.Serialize()));
+    var submitted = taskIds as string[] ?? taskIds.ToArray();
+    foreach (var taskid in submitted)
+    {
+      ResultHandlerDictionary[taskid] = handler;
+    }
+
+    return submitted;
+  }
+
+
+  /// <summary>
+  ///   The method submitAsync will serialize argument in object[] MethodName(object[]  argument).
+  /// </summary>
+  /// <param name="methodName">The name of the method inside the service</param>
+  /// <param name="argument">One serialized argument that will already serialize for MethodName.</param>
+  /// <param name="handler">The handler callBack implemented as IServiceInvocationHandler to get response or result or error</param>
+  /// <param name="token">The cancellation token</param>
+  /// <returns>Returns the taskId string</returns>
+  public async Task<string> SubmitAsync(string                    methodName,
+                                        object[]                  argument,
+                                        IServiceInvocationHandler handler,
+                                        CancellationToken         token = default)
+  {
+    var blockRequest = new BlockRequest
+                       {
+                         Payload = new ArmonikPayload
+                                   {
+                                     ArmonikRequestType  = ArmonikRequestType.Execute,
+                                     MethodName          = methodName,
+                                     ClientPayload       = ProtoSerializer.SerializeMessageObjectArray(argument),
+                                     SerializedArguments = false,
+                                   },
+                         Handler = handler,
+                         Lock    = semaphoreSlim_,
+                       };
+
+    return await SubmitAsync(blockRequest,
+                             token);
```

Comment:
```suggestion
    return await SubmitAsync(methodName,
                             ProtoSerializer.SerializeMessageObjectArray(argument),
                             handler,
                             token).ConfigureAwait(false);
```

Reply:
Can't do that SerializedArguments change between these 2 calls